packages/formik-effect/README.md

```diff
@@ -0,0 +1,76 @@
+# Formik Effect
+
+Declarative Formik component for side-effects. Use with caution. Please. I. beg. you.
+
+```sh
+yarn add @onaio/formik-effect --save
+```
+
+Note: this has peer dependencies of react, and formik.
+
+## The Problem
+
+Formik is an uncontrolled component. However, there are times when you want to trigger a side effect based on a state change. By design, Formik does not expose a prop for you to do this because I'm terrified as to how it would be abused--it encourages people to attempt to "sync" Formik's state in elsewhere (cough like in a Redux store cough cough). Anyways, please don't do that. You never ever ever ever want to store the same state in 2 places in a React application because it is almost impossible to keep them in sync unless you are a goddamn jedi. You may think it's working, and high five a teammate, but you are a just a lifecycle method away from lots and lots of pain and I can guarantee you are not considering all the edge cases. Sooooo....
```

Comment:
This might be the best problem statement ever, of all time 🤣 

"Forget the Action Creators Luke, use the force!" 👋